redfishMockupServer.py

```diff
@@ -394,6 +394,16 @@ def do_GET(self):
                     else:
                         pass
 
+                    # Handle Expand Query
+                    expand = query_pieces.get('$expand', [''])[0]
```

Comment:
It looks like this will just handle expand on collections, but not other types of resources. It also doesn't seem to verify the expand operator (`.`, `*`, or `~`). Specifically for collections, the `~` operator will have no effect since there isn't a `Links` object.

Reply:
Correct. As I wrote in TODO it doesn't handles LEVELS and LOINKSs yet - this is future work

Reply:
Okay, we should probably open an issue to track that once this PR is merged. Otherwise it's good as long as we know the limitation.

Reply:
Thanks @mraineri. I opened #72 to track future work